### ISSUE TITLE
NVDA fails to read out the success message for Messager bar component for MessageBarType=success, but works for type error.

### ISSUE THREAD

**Issue by ApoorvaPetkar** [microsoft/fluentui#18834]
Below is the code I'm using the message bar with tostify.

```
import React from 'react';
import { toast, Flip } from 'react-toastify';
import 'react-toastify/dist/ReactToastify.css';
import { MessageBar, MessageBarType } from '@fluentui/react';
import { MessageType } from '../Constants';
import { ICSSPixelUnitRule } from '@fluentui/merge-styles/lib/IRawStyleBase';
import './NotificationMessage.css';

toast.configure();

interface MessageBarProps {
  message: string;
}
const SuccessMessageBarElement = (props: MessageBarProps) => (
  <MessageBar messageBarType={MessageBarType.success} isMultiline={true}>
    {props.message}
  </MessageBar>
);
const ErrorMessageBarElement = (props: MessageBarProps) => (
  <MessageBar messageBarType={MessageBarType.error} isMultiline={true}>
    {props.message}
  </MessageBar>
);
export const showNotification = (
  message: string,
  messageType: MessageType
): void => {
  if (messageType === MessageType.SUCCESS) {
    toast(<SuccessMessageBarElement message={message} />, {
      className: 'successNotification',
      closeButton: true,
      transition: Flip,
      position: toast.POSITION.TOP_RIGHT
    });
  } else if (messageType == MessageType.ERROR) {
    toast(<ErrorMessageBarElement message={message} />, {
      className: 'errorNotification',
      closeButton: true,
      transition: Flip,
      position: toast.POSITION.TOP_RIGHT
    });
  }
};
```

Actual Behaviour: NVDA screen reader able to read out error message notification, but fails for the success message type.

Excepted Behaviour: NVDA should read out all kinds of Message Bar type.

**Comment by ling1726** [comment]
I have tested the [MessageBar](https://developer.microsoft.com/en-us/fluentui#/controls/web/messagebar) examples with NVDA, the delayed render does work with NVDA and `role="status"` is announced on render.

Can you provide a repro example in codesandbox or codepen, since there is a lot of extra code in your example that could cause this issue such as toasify and custom CSS.

Additionally, you should have the nested live role antipattern in the example that you show. [react-toastify by default will render role="alert"  for all toasts](https://fkhadra.github.io/react-toastify/api/toast) meanwhile, Fluent's `MessageBar` will also have a `status` or `alert` role that's nested inside. Might be interesting to know @smhigley
